packages/material-ui/src/index.d.ts

```diff
@@ -2,6 +2,11 @@ import * as React from 'react';
 import { StyledComponentProps } from './styles';
 export { StyledComponentProps };
 
+/**
+ * Get the type of a component's props.
+ *
+ * @internal
+ */
 export type PropsOf<C> = C extends new (props: infer P) => React.Component
```

Comment:
@pelotom Can we alias this to `React.ComponentProps` now?

Reply:
I had figured we would do that in a separate PR, but no objections to doing it here.